integreat_cms/static/src/css/tinymce_custom.css

```diff
@@ -6,3 +6,7 @@
 [translate="no"] {
     background-color: rgb(239, 68, 68);
 }
+
+.notranslate-invisible {
+    background-color: transparent !important;
```

Comment:
But does this apply here? Phone numbers not being rearranged is a pretty basic expectation (hence this issue 😄). 
I would agree with you if the Workflow was 

- enter pone number
- press space, tinymce formats the number
- user highlights phone number
- presses "do not translate"
- expects phone number to be shown in red, since that's what the user is used to from the DNT button

But since we know that phone numbers should always be DNT, it's only necessary to do
- enter phone number
- press space, tinymce formats number and applies DNT

IMO in the second case, it's weird to have the phone number be formatted and highlighted in red - to me at least, that would indicate that there's something wrong with the phone number

Reply:
I do see your point that the red highlighting is not ideal, but still I was expecting in it and in my previous review this was exactly the issue. I think if the phone number would have been red and therefore marked as DNT I would have understood that it's immediately applied. So I still think we should keep the highlighting, also for phone numbers as I can see municipalities also being confused? But maybe this is more a question for @osmers than for me

Reply:
I did not expect the phone number to be automatically marked as DNT - that's even better I think :D but I would still use some kind of highlighting or similar to let the user know that the section was automatically marked as DNT....Does your current fix still apply DNT automatically upon formatting?

Reply:
> Does your current fix still apply DNT automatically upon formatting?

Yes!

> I would still use some kind of highlighting or similar to let the user know that the section was automatically marked as DNT

Do you have a suggestion? 😅 Or do you agree woth Johannes? In that case I'll re-add the red background

Reply:
The thing is that I am not sure if it's really intuitive that for some things (phone numbers) the DNT is automatic and for others (texts, addresses, etc) it's not - especially if they are then marked differently.
I think I would add back the red highlighting since users already know what it means. This could help in understanding what's happening. Do you think it also makes sense to mark email addresses automatically as DNT?

Also, as soon as we have the contact container and central contact management, we won't really need it anymore anyways, right?

Reply:
> I think I would add back the red highlighting since users already know what it means.

This is the part I am not sure of, because usually red indicates error. In the case of manual DNT, there's a clear "I clikced this button, now the part is highlighted" logic, while just pressing space after typing the phone number could easily be understood as "check this number again, something's not right, this is not valid". At least in my thinking, but I might be alone in that 😅 

> Do you think it also makes sense to mark email addresses automatically as DNT?

I don't think so, unless there's currrently an issue with that. From what I understand, those are always without any spaces, so they can't be rearranged anyways.

> Also, as soon as we have the contact container and central contact management, we won't really need it anymore anyways, right?

Hm... Depends, I assume one-off phone numbers will still be added the old way? 



But in any case, I'll re-add the highlighting for now.

Reply:
Yeah, I follow your thinking :) maybe just bcs the red highlighting is already associated with the DNT function will they understand what's happening. But maybe not :D 
And what if it does not mark it automatically? So we'd only add the keep direction of text functionality and not change anything else? What will lead to more errors/questions?

# Hm... Depends, I assume one-off phone numbers will still be added the old way?
True - and not all municipalities will use the contact management necessarily...

Reply:
> And what if it does not mark it automatically? So we'd only add the keep direction of text functionality and not change anything else? What will lead to more errors/questions?

Or how about this: no-translate is applied automatically, with no highlighting; but I'll make sure that re-applying the DNT manually has no effect besides adding the highlight? AFAIK there's no reason to NOT want the phone numbers to stay LTR, and anyone who expects to DNT them manually can do so.

Reply:
If that works cleanly, that's nice. I'll check with the team on what they prefer bcs I can also see advantages of marking it and making it obvious bcs it saves the municipalities a lot of time...